src/sdk/AvatarSynthesizer.ts

```diff
@@ -156,6 +165,10 @@ export class AvatarSynthesizer extends Synthesizer {
      * @returns {Promise<void>} The promise of the void result.
      */
     public async stopSpeakingAsync(): Promise<void> {
+        while (this.synthesisRequestQueue.length() > 0) {
```

Comment:
Is there a time limit here, or do you have to worry about an infinite wait?

Reply:
That's a good point, let me check

Reply:
This is a corner case, I'd like to merge it first to catch the release.